docker/px4-dev/Dockerfile_ros-kinetic

```diff
@@ -39,9 +39,9 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 	&& apt-get -y autoremove \
 	&& apt-get clean autoclean \
 	# pip
-	&& pip install --upgrade numpy px4tools pymavlink \
-	&& pip3 install --upgrade setuptools wheel \
-	&& pip3 install --upgrade pyulog matplotlib \
+	&& pip install px4tools pymavlink \
+	&& pip3 install setuptools wheel \
```

Comment:
Just to be clear about my motivation here... I didn't see the `--upgrade` flag necessary for a new install. Here is the output from this image:
```
root@72de71e728fa:/# pip3 install --upgrade setuptools wheel
Requirement already up-to-date: setuptools in /usr/local/lib/python3.5/dist-packages
Requirement already up-to-date: wheel in /usr/local/lib/python3.5/dist-packages
```